pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+Please include a summary of the changes, including screenshots, if applicable. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
maybe a bit more here to make explicit that reviewers should have all they need to review from the description

`Please provide context necessary to review these changes`